app/javascript/packs/pw-strength.js

```diff
@@ -18,12 +18,12 @@ const snakeCase = (string) => string.replace(/[ -]/g, '_').replace(/\W/g, '').to
 // fallback if zxcvbn lookup fails / field is empty
 const fallback = ['pw-na', '...'];
 
-function clearErrors() {
-  const x = document.getElementsByClassName('error-message');
-  if (x.length > 0) {
-    x[0].innerHTML = '';
-  }
-}
+// function clearErrors() {
```

Comment:
This function may no longer be necessary. Please correct me if I am mistaken.

Reply:
Yeah, looks to me like it wasn't used for anything. 👍